spa/src/components/common/Modal/AddPageModal/AddPageModal.tsx

```diff
@@ -0,0 +1,114 @@
+import { useMemo } from 'react';
+import AddIcon from '@material-ui/icons/Add';
+import orderBy from 'lodash.orderby';
+import PropTypes, { InferProps } from 'prop-types';
+
+import { Button, MenuItem, Modal, ModalContent, ModalFooter, ModalHeader, TextField } from 'components/base';
+import { Controller, useForm } from 'react-hook-form';
+import { ErrorMessage, Typography } from './AddPageModal.styled';
+
+interface AddPageModalType extends InferProps<typeof AddPageModalPropTypes> {
```

Comment:
For consistency across components...

```suggestion
export interface AddPageModalProps extends InferProps<typeof AddPageModalPropTypes> {
```